mods/gennext/README.md

```diff
@@ -391,6 +391,8 @@ Minor learnset changes:
 
 - Scolipede and Steelix get Coil
 
+- Raichu, Lumineon, Ampharos, Chinchou, Lanturn get Tail Glow
```

Comment:
We never agreed on Raichu :[

Reply:
And prevos like Chinchou aren't mentioned in the README.

Reply:
Chinchou is mentioned lol

Reply:
He meant prevos aren't supposed to be mentioned in the README, but you did mention one, and now even more.

Reply:
Ohhh my bad